src/components/Transactions/Transaction.tsx

```diff
@@ -75,11 +78,7 @@ const Transaction = (props: transactionProps) => {
         </Typography>
       </StyledTableCell>
       <StyledTableCell align="right">
-        <Button
-          variant="naked"
-          onClick={() => navigate(`/transaction/${id}`)}
-          endIcon={<ArrowForwardIosIcon color="primary" />}
-        />
+        <Button variant="naked" endIcon={<ArrowForwardIosIcon color="primary" />} />
```

Comment:
After the mod, this could be a "simple icon". Why use a button if there isn't any action to perform?